README.md

```diff
@@ -336,6 +336,19 @@ settings:
 [`eslint_d`]: https://www.npmjs.com/package/eslint_d
 [`eslint-loader`]: https://www.npmjs.com/package/eslint-loader
 
+## Webpack configuration arguments
+
+This plugin doesn't pass arguments to [Webpack configuration functions](https://webpack.js.org/configuration/configuration-types/#exporting-a-function). If a Webpack configuration function uses the `env` or `argv` arguments, values must be supplied to avoid `argument undefined` errors. On Node 6.4 or greater we recommend [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters):
```

Comment:
is this something that the plugin could fix?

Reply:
Possibly partially.

#### Case A: Using the resolver during compilation

Since webpack and eslint are running in the same env, it could be possible. The relevant webpack function call is here: https://github.com/benmosher/eslint-plugin-import/blob/a69b77181c423043d39f1c8752b296aaa95facfc/resolvers/webpack/index.js#L84-L86

#### Case B: In the code editor

I don't think this case is solvable. There's no way for the editor to know what env you're running your build process in.

Reply:
The plugin could expose a command line or config option to pass args to the webpack function. But, IMO, people have different enough needs that handling in code is probably best, with a simple template provided for the most common case.

I could see is providing a default `env.NODE_ENV` on line 85, to automatically fix the most common case. That would be pretty prescriptive and could cause unexpected behavior, but maybe a good "works out of the box" solution.